spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/RuntimeTemplate.java

```diff
@@ -50,7 +50,9 @@
 
 	@Override
 	public PagedResources<AppStatusResource> status() {
-		return restTemplate.getForObject(appStatusesUriTemplate.expand().getHref(), AppStatusResource.Page.class);
+		String uriTemplate = appStatusesUriTemplate.expand().getHref();
```

Comment:
[Whoever](https://github.com/spring-cloud/spring-cloud-dataflow/blame/master/spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/StreamTemplate.java#L72) did that should be hung :)